pom.xml

```diff
@@ -121,8 +121,10 @@
         <json-path.version>4.1.1</json-path.version>
         <jupiter.version>5.6.2</jupiter.version>
         <junit.platform.version>1.6.2</junit.platform.version>
+        <junit-platform-surefire-provider.version>1.0.3</junit-platform-surefire-provider.version>
```

Comment:
Is there any reason to use this specific version? Latest is 1.3.2

Reply:
I do not think so. I will changed it.